### ISSUE TITLE
New User Can't Login w/Existing Account if Font Size Enlarged Under System Settings

### ISSUE THREAD

**Issue by bobmatyas** [wordpress-mobile/WordPress-iOS#15655]
### Expected behavior

User should be able to login to WordPress.com without needing to reduce font size on device.

### Actual behavior

When installing the Wordpress app on iOS for the first time if a user has their font size increased beyond the system default the user will not see the input box to enter their email for their existing Wordpress.com account.

Instead they will see a screen that says to enter their existing email address but it is text w/no place for the email and the two options to login via Apple or Google below.

Decreasing the system font size under System Settings > Display & Brightness > Text Size will adjust the screen and show an input box allowing the user enter an email and complete the login process.

### Steps to reproduce the behavior

Increase the font size under System  Settings > Display & Brightness > Text Size and the input box will not be visible.

**Comment by kduncanwelke** [comment]
I'm going to investigate both this and the related #15925 as well. 👍🏻

**Comment by kduncanwelke** [comment]
After some investigating, the fix for this seems to trace back to the Authenticator pod, so I will be submitting a PR there, which (once added) should solve the problem.
